pkg/controller/installation/installation_controller.go

```diff
@@ -241,14 +228,13 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 	}
 	metrics.SetRHMIStatus(installation)
 
-	configManager, err := config.NewManager(r.context, r.client, request.NamespacedName.Namespace, installationCfgMap, installation)
+	configManager, err := config.NewManager(context.TODO(), r.client, request.NamespacedName.Namespace, installationCfgMap, installation)
```

Comment:
Any particular reason for no longer using a long lived context?

Reply:
it's stated explicitly that storing a context can cause a memory leak.

Reply:
last paragraph in overview here, for more info: https://golang.org/pkg/context/

Reply:
Oh good point. I think we do this in other places too.